privacyidea/models.py

```diff
@@ -1515,6 +1515,7 @@ class Policy(TimestampMethodsMixin, db.Model):
     active = db.Column(db.Boolean, default=True)
     check_all_resolvers = db.Column(db.Boolean, default=False)
     name = db.Column(db.Unicode(64), unique=True, nullable=False)
+    user_case_insensitive = db.Column(db.Boolean, default=True)
```

Comment:
The current behaviour is, that the username is *casesensitive*.

So the default here for case_insensitive should be "False".
Otherwise we are changing the default behaviour silently.

Reply:
[b7bc726](https://github.com/privacyidea/privacyidea/pull/3852/commits/b7bc726f2279a85bad77918536eb77dbabbf0523)